consoles/console.pm

```diff
@@ -74,10 +74,15 @@ sub screen {
 }
 
 # helper function
+# Keep ssh session for the maximum of ServerAliveCountMax x ServerAliveInterval seconds 
+# without receiving any messsage back from the server, and this will not affect normal
+# ssh disconnect and console switching. Ssh console may not display returned result of 
+# long-time run test with these options. TCPKeepAlive ensures that if network goes down 
```

Comment:
You mean without these options?

Reply:
1. Updated comments 2. ssh connection will be deems as lost during long-time run test on some machines. So no result can be returned after test finishes.